ipynb/Alpha Diversity Regression.ipynb

```diff
@@ -83,28 +91,59 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We're going to only look at samples from the US and UK, since these represent about 92% of the samples in the data set."
+    "Next, we'll load the bloom percentage file, and add that to the mapping file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "bloom = pd.read_csv('./ag-bloom-fractions-125nt.txt', sep='\\t', dtype=str)\n",
+    "bloom.set_index('#SampleID', inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "map_['bloom_fraction'] = bloom['bloom_fraction'].astype(float)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We're going to only look at samples from the US and UK, since these represent about 92% of the samples in the data set. We'll also limit the data based on BMI, limit the data set to participants who have provided their sex and male and female. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 47,
    "metadata": {
     "collapsed": false
    },
    "outputs": [],
    "source": [
     "def filter_map(x):\n",
-    "    country = x['country'] in {'USA', 'United Kingdom', 'Canada'}\n",
+    "    country = x['country'] in {'USA', 'United Kingdom', 'Canada', 'Australia'}\n",
     "    bmi = (x['bmi_corrected'] < 40) and (pd.isnull(x['bmi_corrected']) == False)\n",
     "    sex = x['sex'] in {'male', 'female'}\n",
     "    poo = x['bowel_movement_quality'] not in {\"I don't know, I do not have a point of reference\"}\n",
     "    race = x['race'] in {'Caucasian', 'Asian or Pacific Islander', 'Other'}\n",
-    "    cpn = x['center_project_name'] in {'AG1', 'AG11', 'AG13', 'AG14', 'AG17', 'AG18', \n",
-    "                                       'AG19', 'AG2', 'AG20', 'AG3', 'AG4', 'AG5', \n",
-    "                                       'AG6', 'AG7', 'AG9', 'AGFermentation', \n",
-    "                                       'AGUNDX', 'AGagOffice1', 'AGlost_samples'}\n",
-    "    return country & bmi & sex & poo & race & cpn"
+    "    cpn = x['center_project_name'] in {'AG31', 'AGag35', 'AG27', 'AGag33', 'AGag36', \n",
```

Comment:
Why is there a limiting to specific rounds and why'd that change?

Reply:
rounds are limited based on the sample size after filtering. So, if there are only 3 people in a run prior to filtering, it gets excluded.